oc_config_validate/oc_config_validate/testcases/setget.py

```diff
@@ -83,3 +87,6 @@ def testSetGetJsonCheckCompare(self):
         got = json.loads(self.resp_val)
         cmp, diff = schema.intersectCmp(self.json_value, got)
         self.assertTrue(cmp, diff)
+        if self.set_replace:
+            cmp, diff = schema.intersectCmp(got, self.json_value)
```

Comment:
is this not simply repeating the same as lines 88-89?

Reply:
Well, the parameters are inverted. So when using SetReplace, all elements in the test Json blob MUST be the same as the response, and all the elements in the Response MUST be in the test Json blob. This is an easy way to check that the test and response are exactly the same (in SetUpdate, the comparison is only one way)